products/protractor.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Protractor
+category: framework
+tags: google javascript-runtime test
+iconSlug: protractor
+permalink: /protractor
+versionCommand: npm list protractor
+changelogTemplate: https://github.com/angular/protractor/blob/master/CHANGELOG.md
```

Comment:
This value needs to be corrected. 
The value should have some Tags to replace, like `__LATEST__`, `__CODENAME__`, and so on, to autogenerate each release's `link` property.

According to the [documentation](https://github.com/herodevs/endoflife.date/blob/master/CONTRIBUTING.md#new-adding-a-new-product)

```md
# Template to be used to generate a link for the releases (optional).
# Available variables inside the template are:
# - __RELEASE_CYCLE__: will be replaced by the value of `releaseCycle`,
# - __LATEST__: will be replaced by the value of `latest`,
# - __CODENAME__: will be replaced by the optional `codename`.
# You can even use Liquid Templating inside the template, such as:
#   https://godotengine.org/article/maintenance-release-godot-{{"__LATEST__" | replace:'.','-'}}
# If possible, Do not use a localized URL (such as one containing en-us).
changelogTemplate: "https://link/of/the/__RELEASE_CYCLE__/and/__LATEST__/version"
```

Reply:
Good catch, done

Reply:
Delete `changelogTemplate`

Reply:
done